src/js/disability-benefits/526EZ/pages/veteranInfo.js

```diff
@@ -1,80 +1,64 @@
-import _ from 'lodash';
+import { merge } from 'lodash/fp';
+
+import fullSchema526EZ from 'vets-json-schema/dist/21-526EZ-schema.json';
 
 import currentOrPastDateUI from '../../../common/schemaform/definitions/currentOrPastDate';
 import fullNameUI from '../../../common/schemaform/definitions/fullName';
 import ssnUI from '../../../common/schemaform/definitions/ssn';
 import { genderLabels } from '../../../../platform/static-data/labels';
-import initialData from '../tests/schema/initialData';
 
-import VerifiedReviewContainer from '../components/VerifiedReviewContainer';
+import ReviewCardField from '../components/ReviewCardField';
+
 import {
   veteranInformationViewField,
-  getPage,
   VAFileNumberDescription
 } from '../helpers';
 
-function createVeteranInfoPage(formSchema, isReview) {
-  const { fullName, date } = formSchema.definitions;
+export const uiSchema = {
+  'ui:field': ReviewCardField,
+  'ui:title': 'Veteran information',
+  'ui:options': {
+    viewComponent: veteranInformationViewField
+  },
+  fullName: fullNameUI,
+  socialSecurityNumber: merge(ssnUI, {
+    'ui:title': 'Social Security number (must have this or a VA file number)',
+    'ui:required': form => !form.vaFileNumber
+  }),
+  vaFileNumber: {
+    'ui:title': 'VA file number (must have this or a Social Security number)',
+    'ui:required': form => !form.socialSecurityNumber,
+    'ui:help': VAFileNumberDescription,
+    'ui:errorMessages': {
+      pattern: 'Your VA file number must be between 7 to 9 digits'
```

Comment:
Similar to above, should this be 7 or 9?

Reply:
Yeah, looks like it. This also looks to be this way (wrong) everywhere else I've seen it, too. :grimacing: